includes/modules/also_purchased_products.php

```diff
@@ -13,13 +13,13 @@
 }
 if (isset($_GET['products_id']) && SHOW_PRODUCT_INFO_COLUMNS_ALSO_PURCHASED_PRODUCTS > 0 && MIN_DISPLAY_ALSO_PURCHASED > 0) {
 
-  $also_purchased_products = $db->ExecuteRandomMulti(sprintf(SQL_ALSO_PURCHASED, (int)$_GET['products_id'], (int)$_GET['products_id']), (int)MAX_DISPLAY_ALSO_PURCHASED);
+  $also_purchased_products = $db->ExecuteRandomMulti(sprintf(SQL_ALSO_PURCHASED, (int)$_GET['products_id'], (int)$_GET['products_id'], (int)MAX_DISPLAY_ALSO_PURCHASED));
 
   $num_products_ordered = $also_purchased_products->RecordCount();
 
   $row = 0;
   $col = 0;
-  $list_box_contents = array();
+  $list_box_contents = [];

```

Comment:
Why is this needed?

Reply:
That's a trick question, it isn't needed!